rsrc/strings/specials-text-general.txt

```diff
@@ -185,10 +196,11 @@ Unused
 0 - allow, 1 - prevent
 Unused
 Unused
-0 - don't force, 1 - force if blocked
+Force allow if blocked?
```

Comment:
Kind of confused on the meaning here.

Reply:
It's something like "if the other field is set to 'allow', override the terrain's blockage". I _think_ it only applies when the action being prevented is movement, but not quite sure.

Better suggestions for the wording are welcome.